### DISCUSSION TITLE
Why is the symbol from the linker script resolved to the UnsafeCell<u64> type？

### DISCUSSION THREAD

**Discussion by StrayLittlePunk** [rust-embedded/rust-raspberrypi-OS-tutorials#107] [Tutorial Questions] [answered]
[code](https://github.com/rust-embedded/rust-raspberrypi-OS-tutorials/blob/8ee50a2f8b483d1103f228f2372d0e6f7446ad56/02_runtime_init/src/bsp/raspberrypi/memory.rs#L14)

**Comment by andre-richter** [comment, marked as answer]
Please refer to: https://github.com/rust-lang/nomicon/issues/109

**Reply by StrayLittlePunk** [reply]
thanks, I will check it
